src/eva/time_series/time_series.py

```diff
@@ -0,0 +1,100 @@
+# (C) Copyright 2024- NOAA/NWS/EMC
+#
+# (C) Copyright 2024- United States Government as represented by the Administrator of the
+# National Aeronautics and Space Administration. All Rights Reserved.
+#
+# This software is licensed under the terms of the Apache Licence Version 2.0
+# which can be obtained at http://www.apache.org/licenses/LICENSE-2.0.
+
+
+# --------------------------------------------------------------------------------------------------
+
+
+import numpy as np
+import xarray as xr
+
+
+# --------------------------------------------------------------------------------------------------
+
+
+xr_aggregation_methods = {
+    'mean': lambda ds, dim: ds.mean(dim=dim, skipna=True),
```

Comment:
In this case I would not expect a performance bottleneck from the lambda. While it might be costly if embedded in a larger loop I think here it would be OK and dwarfed by the cost of the actual aggregation method. What kind of cost are you seeing for your case? Could you try some vanilla Python/Xarray that computes the aggregation you want and let me know if the cost is much different than running with Eva?

Reply:
Sorry, I meant when I processed with an offline xarray code that I wrote before. 

Let us implement this in Swell and we can see performance with the full obs system. I don't want to hold back this merge longer since you will be on leave for a while.

Reply:
Ok sounds good.